concent_api/core/views.py

```diff
@@ -554,6 +630,16 @@ def handle_send_force_get_task_result(request, client_message: message.concents.
             requestor_public_key = client_public_key,
             status               = ReceiveStatus,
         )
+        store_or_update_subtask(
+            task_id                     = client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],
+            subtask_id                  = client_message.report_computed_task.task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key         = other_party_public_key,
+            requestor_public_key        = client_public_key,
+            state                       = Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline               = client_message.report_computed_task.timestamp + settings.CONCENT_MESSAGING_TIME,  # TODO: Is it ok ??
```

Comment:
`next_deadline` is wrong here.
- According to the initial blueprint (#12) it was `ReportComputedTask.timestamp` + FAT + CMT.
- When we implement #165 it should be `TaskToCompute.deadline` + 3 CMT + 2 MDT

Reply:
:+1: